### ISSUE TITLE
Finish the merge of the Sprint 1.

### ISSUE THREAD

**Issue by joanbm** [jgirald/ES2015C#37]
Backlog item: N/A

Estimated effort: 2,5 hours

Ending condition: A copy of the project consisting of the integrated work of all teams during sprint 1 must exist in branch 'dev'. All work items of all teams developed during sprint 1 must be present in the result. The scene must contain relevant items developed during the first sprint, and if bugs occur during the integration, they must be reported to the groups responsible for fixing them.

Planned due date: 8 Oct 2015, 23:59 CEST.

**Comment by joanbm** [comment]
I have finished a basic merge of the branches.

All models from team A and team B are visible on the map, and all of them have textures except a few models from team B from which I could not find any image file at all.

All features which were present in the development branches of team C and team D are present in the final result.

A basic consistent folder organization has also been implemented.

I could not get the animations for the civil unit models to work in the final project. I believe that this may be a deeper issue requiring changes to the animation controller or the models so I did not pursue the issue further.
